src/types.ts

```diff
@@ -647,6 +643,12 @@ export interface OutputItemsPayload {
   id: string
 }
 
+export interface AnnotationsPayload {
```

Comment:
Perhaps moving this up to be more colocated with where's being used. Seems odd so far down since annotations are so integral to Runme.

Reply:
Good point. I moved it up close to the previous one. Let me move this to a better place.